### PR TITLE
New Bikehorn for Toodles

### PR DESCRIPTION
**Pull request by Terilia**
ckey:Xerdies
Player: Toodles
Fluff Type: New bikehorn for Toodles
Icon: 
Left Hand: 
Right Hand: 

Note: Dumb made those for me and I am uncertain how to display them better ><

🆑 Terilia
Fluff: Added a new bikehorn for Toodles to ~~entertain~~ prank people with. 
/🆑

### CODE REVIEW

**Comment by Spartan6** [comment]
Need sprite pictures for the actual item too, not just in-hand.

**Comment by Terilia** [comment]
Huh? is it missing in the files? The picture up there is just an example pic. But I think all needed files are in the textures.

**Comment by Spartan6** [comment]
You need to show pictures of both the in hand and the floor sprite in the opening post.

**Comment by Terilia** [comment]
Thanks! Done

**Comment by Necaladun** [comment]
:+1:

File: code/modules/customitems/item_defines.dm [hunk 1]
```diff
@@ -1262,3 +1262,13 @@
 	item_state = "victorianvest"
 	item_color = "victorianlightfire"
 	displays_id = FALSE
+
+/obj/item/weapon/bikehorn/fluff/pinkbikehorn
```
Comment (tigercat2000) [inline code/modules/customitems/item_defines.dm]: Add `//Ckey: Character Name` like the above fluff items please

**Comment by Terilia** [comment]
Ok done :)

**Comment by Regen1** [comment]
Are you sure that's not a penis

**Comment by Terilia** [comment]
Yes .-. very sure

**Comment by taukausanake** [comment]
> Are you sure that's not a penis

That time when we had [a dildo PR](https://github.com/ParadiseSS13/Paradise/pull/3333) open

**Comment by Fox-McCloud** [comment]
conflicts; as soon as these are resolved, this can be merged.

**Comment by Fox-McCloud** [comment]
please resolve your conflicts.

**Pull request merged**